### ISSUE TITLE
Hope to support react-native 0.56

### ISSUE THREAD

**Issue by fzdm** [nihgwu/react-native-dva-starter#64]
Hi,There are many problems in the current version of react-native upgrade version 0.56.

**Comment by gatspy** [comment]
i have upgraded successfully.there are three steps:
1. upgrade with [react-native-git-upgrade](https://github.com/facebook/react-native/tree/master/react-native-git-upgrade)
```bash
react-native-git-upgrade 0.56.0
```
2. modify package.json
```json
{
    "devDependencies": {
        "@babel/plugin-proposal-decorators": "7.0.0-beta.47",
        "babel-preset-react-native": "^5.0.0",
    }
}
```
3. run app
```bash
rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build && rm -rf node_modules/ && npm cache clean --force && yarn install

react-native run-ios
```

**Comment by qwlong** [comment]
@gatspy 
Good job, It saves me a lot of time.

**Comment by nihgwu** [comment]
closed via #76
